src/main/java/org/sagebionetworks/web/client/DisplayUtils.java

```diff
@@ -644,7 +643,7 @@ public static String getSynapseHistoryToken(
     String entityId,
     Long versionNumber
   ) {
-    return "#" + getSynapseHistoryTokenNoHash(entityId, versionNumber);
+    return getSynapseHistoryTokenNoHash(entityId, versionNumber);
   }
 
   public static String getSynapseHistoryTokenNoHash(
```

Comment:
Since we're completely removing the hash, I would expect each of the `getSynapseHistoryTokenNoHash` to be either renamed or deleted so we just have `getSynapseHistoryToken` methods (unless these are somehow useful to redirect traffic from `#!` URLs)

Reply:
Good call